waiter/src/waiter/process_request.clj

```diff
@@ -502,7 +501,7 @@
              :status 303})
           (do
             (meters/mark! (metrics/waiter-meter "core" "process-errors"))
-            (utils/exception->response e request)))))))
+            (utils/exception->response error request)))))))
```

Comment:
The two versions are no longer equivalent as we will not mark `process-errors` if the `handler` call throws an exception.